overlay-with-config.nix

```diff
@@ -25,12 +25,14 @@ final: prev: (
         if cfg.som == null then null
         else if lib.hasPrefix "orin-" cfg.som then "t234"
         else if lib.hasPrefix "xavier-" cfg.som then "t194"
+        else if lib.hasPrefix "thor-" cfg.som then "t264"
         else throw "Unknown SoC type";
 
       chipId =
         if cfg.som == null then null
         else if lib.hasPrefix "orin-" cfg.som then "0x23"
         else if lib.hasPrefix "xavier-" cfg.som then "0x19"
+        else if lib.hasPrefix "thor-" cfg.som then "0x26"
```

Comment:
question: where can I find these magic numbers myself?

Out of curiosity, I wasn't able to find these in NVIDIA's docs. But there's a high chance I was looking in the wrong places. 😅